src/dataLoaders/actions/dataLoaders.ts

```diff
@@ -395,7 +396,8 @@ export const createOrUpdateTelegrafConfigAsync = () => async (
     },
   }
 
-  const plugins = telegrafPlugins.reduce(
+  // TODO: convert this into a generic type
```

Comment:
Can we update this TODO with a link to an issue so we follow-up on this typecasting?